### DISCUSSION TITLE
Active hours

### DISCUSSION THREAD

**Discussion by wahlstedt** [Geektoolkit/Dynaframe3#64] [Ideas]
This project is coming along nicely! One feature I'm missing is the ability to automatically control when the screen should be active during the day. So that for example, the screen would turn on at 8am and turn off at 7pm. This way the screen isn't used unnecessarily when nobody is around. (Future updates could maybe use an ambient sensor to activate the screen when the room lights turns on.)

**Comment by jorwern** [comment]
https://www.thedigitalpictureframe.com/how-to-automatically-turn-your-digital-picture-frame-on-and-off-at-fixed-times/

**Reply by wahlstedt** [reply]
@jorwern Thanks, that's helpful. I'll definitely use that until Dynaframe gets built-in support for that functionality. 👍🏼

**Reply by jorwern** [reply]
https://www.thedigitalpictureframe.com/start-systemd-services-realtime-timer/ Systemd is even more convient

**Reply by Geektoolkit** [reply]
I think short to mid term that would be the preferred way to do it, though I can see that some basic automation built into the frame would be handy.  The frame is very hacker friendly in that it exposes just about all of it's functionality through http APIs which are documented.  So you could add a motion sensor, use python when the person has left to pause the slide show and turn off the screen, and then resume and turn it on when they return.

Basic on/off at certain times though makes sense. I'll look into that.
